CONTRIBUTING.md

```diff
@@ -1,28 +1,7 @@
-# Contributing
+# Contributors Manual
```

Comment:
`Contributor Manual`? `Contributor’s Manual`?

Reply:
from a quick google the singular form seems more popular: https://www.google.com/search?q=contributor+manuel